docs/rules/arazzo/sourceDescriptions-empty.md

```diff
@@ -0,0 +1,119 @@
+# sourceDescriptions-empty
```

Comment:
This rule doesn't really make sense. 

https://spec.openapis.org/arazzo/latest.html#arazzo-specification-object

`sourceDescriptions` is required as part of the spec.

Reply:
Yes, it is required, but in that case you can`t use only `x-operation` without adding some dummy sourceDescription.
I think we need to have this rule and it will allow you to omit sourceDescription if you don't have any, or will tell you to add one if `opearationId` | `oprationPath` is used.

Reply:
Alternative solution might be allow `sourceDescriptions` to be empty list.
But this will be against Arazzo spec as it should have at least one described item.

Reply:
Discussed offline  with @adamaltman ->  I will transform this rule to check that there is at least one sourceDescription as per Arazzo Spec.
There will be no way to use Spot without description added to sourceDescriptions.